routes/api/gameApi.js

```diff
@@ -29,11 +28,11 @@ router.get("/", withAuth, async (req, res) => {
       res.json(games);
     } catch (e) {
       console.log(e);
-      return res.status(409).json({ message: `The title already exists.` });
+      return res.status(409).json({ message: `Unexpected Error!` });
```

Comment:
If this is an error, 409 is the wrong status code. 409 means unable to update.

Reply:
will be fixed to 

      return res.status(500).json({ message: "Unexpected Error!" });